packages/manager/modules/exchange/src/account/add/account-add.html

```diff
@@ -17,363 +14,908 @@
+                                class="oui-field__helper"
+                                data-ng-if="$ctrl.accountCreationOptions.passwordComplexityEnabled"
+                            >
+                                <span
+                                    data-ng-bind=":: 'exchange_account_add_password_hint_complex_1' | translate"
+                                ></span>
+                                <ul>
+                                    <li
+                                        data-ng-bind=":: 'exchange_account_add_password_hint_complex_1_1' | translate"
+                                    ></li>
+                                    <li
+                                        data-translate="exchange_account_add_password_hint_complex_1_2"
+                                        data-translate-values="{ t0: $ctrl.accountCreationOptions.minPasswordLength }"
+                                    ></li>
+                                    <li>
+                                        <span
+                                            data-ng-bind=":: 'exchange_account_add_password_hint_complex_1_3' | translate"
+                                        ></span>
+                                        <ul>
+                                            <li
+                                                data-ng-bind=":: 'exchange_account_add_password_hint_complex_1_3_1' | translate"
+                                            ></li>
+                                            <li
+                                                data-ng-bind=":: 'exchange_account_add_password_hint_complex_1_3_2' | translate"
+                                            ></li>
+                                            <li
+                                                data-ng-bind=":: 'exchange_account_add_password_hint_complex_1_3_3' | translate"
+                                            ></li>
+                                            <li
+                                                data-ng-bind=":: 'exchange_account_add_password_hint_complex_1_3_4' | translate"
+                                            ></li>
+                                        </ul>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+
+                        <!--Confirm password-->
+                        <div
+                            class="col-md-6 oui-field"
+                            data-ng-class="{ 'oui-field_error': $ctrl.newAccountForm.passwordConfirmation.$invalid && $ctrl.newAccountForm.passwordConfirmation.$touched }"
+                        >
+                            <label
+                                for="passwordConfirmation"
+                                class="oui-field__label oui-label"
+                                data-ng-bind=":: 'exchange_account_add_confirm_password_label' | translate"
```

Comment:
**nitpick(non-blocking):** We can replace `ngBind` directive + `translate` filter by a `translate` directive.